alibi_detect/saving/typing.py

```diff
@@ -0,0 +1,41 @@
+from abc import ABC
```

Comment:
Would be good to add a docstring about the purpose of this module and the `ConfigDetector` class.